gapstool.vbs

```diff
@@ -10,17 +10,24 @@
 ' distributed under the License is distributed on an "AS IS" BASIS,
 ' WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 ' See the License for the specific language governing permissions and
-' limitations under the License.

+' limitations under the License.
+
 ' GAPS diagnostics tool
 ' Liron Newman lironn@google.com
 
 ' Do not change this line's format, build.bat relies on it.
-Const Ver = "1.0.1.0"

+Const Ver = "1.0.1.0"

```

Comment:
Why is this showing as changed? This and other places. Maybe your branch is out of date (not synced to master or whatnot)?